drivers/amazonec2/amazonec2.go

```diff
@@ -519,29 +521,6 @@ func (d *Driver) sshKeyPath() string {
 	return path.Join(d.storePath, "id_rsa")
 }
 
-func (d *Driver) updateDriver() error {
```

Comment:
This didn't seem to be doing anything which wasn't already done by the main `Create` function or the other check for IP address, so I've taken it out.  Instance creation still works fine, so let me know if I'm overlooking something.